pages/common/git-rebase.md

```diff
@@ -27,3 +31,7 @@
 - Reapply the last 5 commits in-place, stopping to allow them to be reordered, omitted, combined or modified:
 
 `git rebase -i {{HEAD~5}}`
+
+- Auto-resolve any conflicts by favoring the working branch version (`theirs` keyword has reversed meaning in this case). See https://git-scm.com/docs/merge-strategies for more information:
```

Comment:
Not sure about the `See <url> for more information.` here.

Reply:
Yeah, we don't really use this pattern except in the main description, and it makes the line way too long IMO.